packages/rich-text/src/apply-format.js

```diff
@@ -17,8 +17,8 @@ import { normaliseFormats } from './normalise-formats';
  *
  * @param {Object} value      Value to modify.
  * @param {Object} format     Format to apply.
- * @param {number} startIndex Start index.
- * @param {number} endIndex   End index.
+ * @param {?number} startIndex Start index.
```

Comment:
I think anywhere there's a change in inline docs, the indentation was not adjusted. :)